bee-message/tests/parents.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::{parents::Parent, prelude::*};
+use bee_packable::{Packable, UnpackError};
+
+// TODO: Add MessageId functions to bee-test
```

Comment:
Still TODO ?

Reply:
Yeah, we can add those into `bee-test` once this is merged 🙂

Reply:
What functions are you speaking about ?